app/code/community/Inviqa/SymfonyContainer/Model/Observer.php

```diff
@@ -12,9 +12,15 @@ class Inviqa_SymfonyContainer_Model_Observer
     public function onCacheRefresh(Varien_Event_Observer $event)
     {
         if (ConfigurationBuilder::MODEL_ALIAS === $event->getType()) {
-            $filePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER;
-            if (file_exists($filePath)) {
-                unlink($filePath);
+            $containerFilePath = Mage::getBaseDir('cache') . '/' . ConfigurationBuilder::CACHED_CONTAINER;
```

Comment:
@adragus-inviqa I was also thinking we should be using DIRECTORY_SEPARATOR, doesn't file_exists() involve reading?